modules/random/random.go

```diff
@@ -24,7 +24,7 @@ func RandomString(elements []string) string {
 	return elements[index]
 }
 
-const base62chars = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz"
+const base62chars = "0123456789012345678901234567890123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz"
```

Comment:
This seems like an unneeded change, since it is replicating the digits.

Reply:
Reverted. The frequency of numbers returned was just low since there is a 5:26 ratio of numeric to alpha chars, tried to even the odds.